common/app/model/content.scala

```diff
@@ -175,15 +175,15 @@ final case class Content(
 
   lazy val linkCounts = LinkTo.countLinks(fields.body) + fields.standfirst.map(LinkTo.countLinks).getOrElse(LinkCounts.None)
 
-  lazy val hasMultipleVideosInPage: Boolean = mainVideoCanonicalPath match {
-    case Some(_) => numberOfVideosInTheBody > 0
-    case None => numberOfVideosInTheBody > 1
-  }
+  lazy val mainMediaVideo = Jsoup.parseBodyFragment(fields.main).body.getElementsByClass("element-video")
 
-  lazy val mainVideoCanonicalPath: Option[String] = Jsoup.parseBodyFragment(fields.main).body.getElementsByClass("element-video").headOption.map { v =>
-    new URL(v.attr("data-canonical-url")).getPath.stripPrefix("/")
+  lazy val hasMultipleVideosInPage: Boolean = mainMediaVideo.isEmpty match {
+    case true   => numberOfVideosInTheBody > 1
+    case false  => numberOfVideosInTheBody > 0
   }
 
+  lazy val mainVideoCanonicalPath: String = new URL(mainMediaVideo.attr("data-canonical-url")).getPath.stripPrefix("/")
```

Comment:
This will blow up if `mainMediaVideo` is empty:

```
scala> new java.net.URL("")
java.net.MalformedURLException: no protocol:
  at java.net.URL.<init>(URL.java:586)
```

So maybe it should be `Option[String]`? e.g.

```scala
lazy val mainVideoCanonicalPath: Option[String] = mainMediaVideo.attr("data-canonical-url") match {
  case "" => None
  case url => Some(new URL(url).getPath.stripPrefix("/"))
}